API/Routes/index.js

```diff
@@ -3,9 +3,12 @@ import IndexRouter from './indexRouter';
 import HomepageRouter from './HomePageRouter';
 import AllmemoriesRouters from './AllMemoriesRouters';
 import AllMemoriesControllers from '../Controllers/AllMemoriesControllers';
-
+import ViewMemoryController from '../Controllers/ViewMemoryController';
+import ViewMemoryRouter from './ViewMemoryRouter';
 
 const index = express.Router();
+
 // merge IndexRouter into index
-index.use(AllmemoriesRouters, HomepageRouter, IndexRouter,);
+index.use(AllmemoriesRouters, HomepageRouter,ViewMemoryRouter, IndexRouter,);
```

Comment:
```suggestion
index.use(AllmemoriesRouters, HomepageRouter,ViewMemoryRouter, IndexRouter);
```
Please remove that last `,`

Reply:
I don't understand what you mean by remove the last, it's not two there

Reply:
The last comma before you close the parentheses.
![Screenshot 2021-07-22 at 21 34 26](https://user-images.githubusercontent.com/30579559/126706021-5dc92eee-c8b5-4343-8f02-bcda17b7d078.png)